.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-*                                    @MetaMask/devs
+*                                   @MetaMask/shared-libraries-engineers
```

Comment:
```suggestion
*                                   @MetaMask/shared-libraries-engineers

```

Reply:
Fixed in [6d4d99c](https://github.com/MetaMask/eslint-config/pull/343/commits/6d4d99cb353485899dc9b4b7b5c8dc571ac81b9d)